.github/workflows/tests.yml

```diff
@@ -61,6 +61,9 @@ jobs:
 
       - name: Test (no default features)
         run: cargo test --no-default-features
+      
+      - name: Test (all features)
+        run: cargo test --all-features
```

Comment:
The `python` feature can only be built with the PyO3 tools. So this line needs to be all of the features, minus the `python` feature.

Reply:
Apparently --no-default-features only tests "std" (that is my understanding)
Maybe we should create a dedicated step like "ANS1" ? I don't know

Reply:
Yes, that would also work